docs/cert-compat.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Certificate Compatibility
+permalink: /docs/certificate-compatibility/
+top_graphic: 1
+---
+
+[<- Back to Documentation List](/docs/)
+
+This page contains lists of software which are known to be able or unable to validate Let's Encrpyt certificates.
```

Comment:
Maybe link "whether the platform supports modern SHA-2 certificates" to https://support.globalsign.com/customer/portal/articles/1499561-sha-256-compatibility#1a.